pkg/docker-compose/build.go

```diff
@@ -8,14 +8,9 @@ const dockerComposeVersion = "1.26.0"
 
 // Build installs the docker and docker-compose binaries
 func (m *Mixin) Build() error {
-	dockerfileLines := fmt.Sprintf(`ENV DOCKER_VERSION="19.03.8"
-RUN apt-get update && apt-get install -y python3-pip wget && pip3 install --upgrade pip && \
-  wget https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz && \
-  tar -xvf docker-${DOCKER_VERSION}.tgz && \
-  mv docker/docker /usr/bin/docker && \
-  chmod +x /usr/bin/docker && \
-  rm -rf docker/ docker-${DOCKER_VERSION}.tgz && \
-  pip3 install docker-compose==%s`, dockerComposeVersion)
+	dockerfileLines := fmt.Sprintf(`RUN apt-get update && apt-get install -y curl && \
+curl -L "https://github.com/docker/compose/releases/download/%s/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
```

Comment:
Can we remove the calls to uname and replace them with linux / amd64 since that isn't variable for mixins?

Reply:
Looks like debian:stretch-slim returns x86_64 for `uname -m`, so I used this instead.  (`amd64`, although it downloaded fine from docker-compose, interestingly resulted in exec format error at runtime)